backend/packages/Upgrade/src/init/seed/initMetrics.ts

```diff
@@ -1,12 +1,19 @@
 import Container from 'typedi';
+import { SERVER_ERROR } from 'upgrade_types';
 import { MetricService } from '../../api/services/MetricService';
 import { env } from '../../env';
 
 export function InitMetrics(): Promise<any> {
   const metricService: MetricService = Container.get(MetricService);
   // Init default metrics in system
   if (env.initialization.metrics) {
-  return metricService.saveAllMetrics(env.initialization.metrics);
+    try {
+      return metricService.saveAllMetrics(JSON.parse(env.initialization.metrics));
+    } catch (error) { 
+      error = new Error('Error while initializing metrics');
+      (error as any).type = SERVER_ERROR.QUERY_FAILED;
```

Comment:
This is not a query failed error.
It's not necessary to assign type to all the errors. The error with type is shown on the UI. So remove the type if we don't have any appropriate error code.

Reply:
Okay. This can be due to JSON parsing or QUERY_FAILED ON DB. We dont have type for JSON parsing error, so have used QUERY_FAILED. Ok, I will remove the type.